input/ignoreWarnings.txt

```diff
@@ -4,3 +4,7 @@
 The HTML fragment 'ip-statements.xhtml' is not included anywhere in the produced implementation guide
 An HTML fragment from the set [cross-version-analysis.xhtml, cross-version-analysis-inline.xhtml] is not included anywhere in the produced implementation guide
 The HTML fragment 'globals-table.xhtml' is not included anywhere in the produced implementation guide
+
+# === These warnings are suppressed as the validator cannot currently fetch the NCTS Codesystem and ValueSet profiles ===
```

Comment:
CI build does not ignore warnings (couple of exceptions) as that allows for continuous QA of both the IG and the toolchain. The release branches will ignore warnings.

What should be added, if these will not be fixed is an editorial note on how and when this will be fixed potentially with reference to a Jira or GitHub or zulip or some other tracker that provides evidence a fix is progressing..

Reply:
Updated suppression comment as per feedback